arch/x86/kernel/cpu/resctrl/monitor.c

```diff
@@ -640,7 +669,7 @@ void cqm_handle_limbo(struct work_struct *work)
 
 	__check_limbo(d, false);
 
-	if (has_busy_rmid(r, d))
+	if (has_busy_rmid(d))
```

Comment:
At upstream patch there is 2 lines removed related to rdt_resource r value.

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev